### PR TITLE
Fix the long description of istio-iptables shown in the 'help' output

### PR DESCRIPTION
**Pull request by jimmy-zh**
istio-iptables is now a binary instead of a script, so we should fix its long description shown in the "help" output.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[X] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by jimmy-zh** [comment]
@googlebot I signed it!

File: tools/istio-iptables/pkg/cmd/root.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ var (
 var rootCmd = &cobra.Command{
 	Use:   "istio-iptables",
 	Short: "Set up iptables rules for Istio Sidecar",
-	Long:  "Script responsible for setting up port forwarding for Istio sidecar.",
+	Long:  "Binary responsible for setting up port forwarding for Istio sidecar.",
```
Comment (hzxuzhonghu) [inline tools/istio-iptables/pkg/cmd/root.go:46]: istio-iptables is responsible for setting up port forwarding for Istio sidecar.

Comment (rlenglet) [reply tools/istio-iptables/pkg/cmd/root.go:46]: Or better: `istio-iptables sets up the port forwarding the Istio Sidecar.`

Comment (jimmy-zh) [reply tools/istio-iptables/pkg/cmd/root.go]: > istio-iptables is responsible for setting up port forwarding for Istio sidecar.

Done

**Comment by jimmy-zh** [comment]
@hzxuzhonghu I have corrected the PR, please review again. Thank you!

**Review by howardjohn — APPROVED** [review]
/ok-to-test

**Pull request merged**